iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_sender.hpp

```diff
@@ -32,6 +33,9 @@ enum class AllocationError
     TOO_MANY_CHUNKS_ALLOCATED_IN_PARALLEL
 };
 
+struct BasePortData;
```

Comment:
Ugly dependency I would say. No problem with the forward declaration but not nice. There is no ports in ChunkSender

Reply:
solved